commit 9603dc29fecabba7fcda520983c8c5d28d946b04
Author: Justin Maggard <jmaggard10@gmail.com>
Date:   Tue Aug 5 14:50:44 2014 -0700

    Bump up idle timeout value, so we don't keep restarting the server unnecessarily.

diff --git a/server/snapperd.cc b/server/snapperd.cc
index 6e6152d..a9a81b8 100644
--- a/server/snapperd.cc
+++ b/server/snapperd.cc
@@ -37,8 +37,8 @@
 using namespace std;
 
 
-const int idle_time = 60;
-const int snapper_cleanup_time = 30;
+const int idle_time = 120;
+const int snapper_cleanup_time = 60;
 
 bool log_stdout = false;
 bool log_debug = false;
